diff --git a/registry.js b/registry.js
index 02470da3c4962ad1bbdc62d9ed295c19ca4905fe..8bf537f37f2d0b877dcf57bf38e17dda3edb47b0 100644
--- a/registry.js
+++ b/registry.js
@@ -8,31 +8,18 @@
  * @format
  */
 
-'use strict';
+"use strict";
 
-export type PackagerAsset = {
-  +__packager_asset: boolean,
-  +fileSystemLocation: string,
-  +httpServerLocation: string,
-  +width: ?number,
-  +height: ?number,
-  +scales: Array<number>,
-  +hash: string,
-  +name: string,
-  +type: string,
-  ...
-};
+const assets = [];
 
-const assets: Array<PackagerAsset> = [];
-
-function registerAsset(asset: PackagerAsset): number {
+function registerAsset(asset) {
   // `push` returns new array length, so the first asset will
   // get id 1 (not 0) to make the value truthy
   return assets.push(asset);
 }
 
-function getAssetByID(assetId: number): PackagerAsset {
+function getAssetByID(assetId) {
   return assets[assetId - 1];
 }
 
-module.exports = {registerAsset, getAssetByID};
+module.exports = { registerAsset, getAssetByID };
